src/OSPSuite.Core/Domain/Mappers/ProcessRateParameterCreator.cs

```diff
@@ -49,27 +50,27 @@ public IParameter CreateProcessRateParameterFor(IProcessBuilder processBuilder,
          return parameter;
       }
 
-      private void addAdditionalParentReference(IFormula formula)
-      {
-         foreach (var objectPath in formula.ObjectPaths)
-         {
-            if (isRelativePath(objectPath))
-               objectPath.AddAtFront(ObjectPath.PARENT_CONTAINER);
-         }
-      }
+      private void addAdditionalParentReference(IFormula formula, string processName) => formula.ObjectPaths.Each(x => adjustRelativePath(x, processName));
 
-      private bool isRelativePath(ObjectPath objectPath)
+      private void adjustRelativePath(ObjectPath objectPath, string processName)
       {
          if (!objectPath.Any())
-            return false;
-
-         if (objectPath[0] == ObjectPath.PARENT_CONTAINER)
-            return true;
+            return;
 
-         if (objectPath.Count == 1)
-            return true;
+         // if the path starts with ".."  or if the objectPath only has the name of the reference then it
+         // should be adjusted to have an additional ".." at the front
+         if (objectPath[0] == ObjectPath.PARENT_CONTAINER || objectPath.Count == 1)
+         {
+            objectPath.AddAtFront(ObjectPath.PARENT_CONTAINER);
+            return;
+         }
 
-         return false;
+         // if the path contains two elements and the first one is the process name then we need to move up two containers at the front
+         // to compensate. That's because a container name can be ignored when a path is resolved from within that container
+         if (objectPath.Count == 2 && string.Equals(objectPath[0], processName))
+         {
+            objectPath.AddAtFront(new ObjectPath(ObjectPath.PARENT_CONTAINER, ObjectPath.PARENT_CONTAINER));
```

Comment:
@Yuri05 what do you think? should we make this case `..|..|ProcessName|ParameterName` or remove the process name and go with `..|ParameterName`

Reply:
`..|ParameterName` would be better for sure (and process name invariant)